### PR TITLE
Sample Comparison Report Testing

### PR DESCRIPTION
**Pull request by labkey-tchad**
Add functionality to assay and sample set API helpers

### CODE REVIEW

File: src/org/labkey/test/Locators.java [hunk 1]
```diff
@@ -21,7 +21,6 @@
     public static final Locator.IdLocator folderMenu = Locator.id("folderBar");
     public static final Locator.XPathLocator labkeyError = Locator.byClass("labkey-error");
     public static final Locator.XPathLocator labkeyMessage = Locator.byClass("labkey-message");
-    public static final Locator.XPathLocator alertWarning = Locator.byClass("alert").withClass("alert-warning");
```
Comment (labkey-danield) [inline src/org/labkey/test/Locators.java:24]: Just curious why this is being removed.

**Review by labkey-danield — COMMENTED** [review]
Looks ok to me.

File: src/org/labkey/test/Locators.java (hunk 1 above)
Comment (labkey-tchad) [reply src/org/labkey/test/Locators.java:24]: It was unused and is probably redundant with `BootstrapLocators.warningBanner`

**Review by labkey-chrisj — APPROVED** [review]
this looks good.  Good call using createSampleSetApi instead of createDomain for createEmptySampleSet()

**Review by labkey-danield — APPROVED** [review]
I'm ok with this, but it might be worth while to run this branch in the Sample Management Sample and SampleSet suites:
https://teamcity.labkey.org/buildConfiguration/LabKey_Trunk_LabkeyPremiumTrunk_GitModules_SampleManagementSamplesAndSampleSetsPostgres?branch=
https://teamcity.labkey.org/buildConfiguration/LabKey_Trunk_LabkeyPremiumTrunk_GitModules_SampleManagementSamplesAndSampleSetsSqlserver?branch=
The API helper is used in those tests.

**Pull request merged**